src/python/grpcio/grpc/__init__.py

```diff
@@ -266,6 +266,22 @@ class StatusCode(enum.Enum):
     UNAUTHENTICATED = (_cygrpc.StatusCode.unauthenticated, 'unauthenticated')
 
 
+#############################  gRPC Status  ################################
+
+
+class Status(six.with_metaclass(abc.ABCMeta)):
```

Comment:
We briefly discussed having accessors on this class so that we could use either `get_details` or `get_message`. Do we still want to do that?

Reply:
To make the API design consistent with `HandlerCallDetails`/`ClientCallDetails`/`RpcError`, IMO we shouldn't add getter/setter here.